scalafix-core/shared/src/main/scala/scalafix/internal/rule/NoAutoTupling.scala

```diff
@@ -7,7 +7,10 @@ import scalafix.rule.RuleCtx
 import scalafix.rule.SemanticRule
 
 case class NoAutoTupling(index: SemanticdbIndex)
-    extends SemanticRule(index, "NoAutoTupling") {
+    extends SemanticRule(
+      index,
+      "NoAutoTupling",
+      "Rule that rewrites tuples defined without parenthesis in parenthesis-wrapped tuples") {
```

Comment:
Rewrite that inserts explicit tuples for adapted argument lists for compatibility with -Yno-adapted-args